### ISSUE TITLE
Validation errors not displaying since Nova 3.2.0

### ISSUE THREAD

**Issue by adriaanzon** [whitecube/nova-flexible-content#162]
After updating to Nova 3.2.0, flexible fields stopped displaying validation errors.

Nova does show the toast alert with "There was a problem submitting the form", but the flexible fields with errors don't have a red border nor a validation message.

Nova 3.2.0 includes changes to validation:
* https://github.com/laravel/nova-issues/issues/1962
* https://github.com/laravel/nova/pull/836
* https://github.com/laravel/nova-js/commit/4fe3c28683f20db485d7ddc50df579c63c6ad2ed

**Comment by voidgraphics** [comment]
Thanks for reporting and including that change list. That's a big help. We'll get it fixed asap.

**Comment by mikaelpopowicz** [comment]
Hi @Nyratas @voidgraphics 

Since laravel/nova-js was tagged v1.2.1 : the mixin **HandlesValidationErrors** changed.

```diff
export default {

     // ...

+    validationKey() {
+      return this.field.validationKey
+    },
    hasError() {
-      return this.errors.has(this.fieldAttribute)
+      return this.errors.has(this.validationKey)
    },
}
```

It can be fixed by refactoring the Group's renameFields method:
```diff
export default class Group {

    // ...

    renameFields() {
        for (var i = this.fields.length - 1; i >= 0; i--) {
            this.fields[i].attribute = this.key + '__' + this.fields[i].attribute;
+           this.fields[i].validationKey = this.fields[i].attribute;
        }
    }
}
```

Hope it helps.

**Comment by ngiraud** [comment]
Hi @voidgraphics, and thanks for the package! Could you please tag the changes so I didn't have to get from the master branch?

**Comment by voidgraphics** [comment]
Hello @ngiraud! Okay, I just published `v0.2.3`.
